### ISSUE TITLE
Can't set a description/body for a marker to be passed to the info-window

### ISSUE THREAD

**Issue by clawfire** [huafu/ember-google-map#81]
Maybe the [documentation](https://github.com/huafu/ember-google-map/wiki/Examples) isn't up to date but as states in it I try to use this object as marker : 

```javascript
{title: "Home", lat: 14.766127, lng: 102.810987, body: "Here is B&H's home"}
```

but the body isn't passed to the `info-window` created. I tried to rename the `body` property in `description` as I saw code using this instead in the issues but without any success. 

any idea ?

**Comment by brianschmitt** [comment]
I was having the same issue, turns out when this was updated to use Controller in place of ObjectController, you also have to modify the fields in the template:
http://emberjs.com/deprecations/v1.x/#toc_objectcontroller

So the proper way to reference the fields in the template is now:
{{model.body}}

**Comment by huafu** [comment]
A fix has been merged and will be deployed in 0.0.24 in a few minutes, the `description` wasn't aliased
